src/lib/Components/Bidding/Screens/__tests__/ConfirmBid-tests.tsx

```diff
@@ -821,10 +841,13 @@ const saleArtwork = {
     },
   },
   sale: {
-    id: "best-art-sale-in-town",
-    endsAt: "2018-05-10T20:22:42+00:00",
+    slug: "best-art-sale-in-town",
+    live_start_at: "2018-05-09T20:22:42+00:00",
+    end_at: "2018-05-10T20:22:42+00:00",
   },
   lot_label: "538",
+  " $fragmentRefs": null, // needs this to keep TS happy
+  " $refType": null, // needs this to keep TS happy
```

Comment:
I wasn't sure how I could keep TS happy without specifying these two types, but I think there might be a good way to do so. cc @alloy

Reply:
Specifying null is fine 👍